mad-dog/src/tests/test_userReplicaSetNodes/index.js

```diff
@@ -0,0 +1,58 @@
+const { addAndUpgradeUsers } = require('../../helpers.js')
+const verifyValidCNs = require('./verifyValidCN')
+const deregisterCreatorNode = require('./deregisterCreatorNode.js')
+const stopRandomCreatorNode = require('./stopRandomCreatorNode.js')
+const setNumCreatorNodes = require('./setNumCreatorNodes.js')
+
+/**
+ * Tests that the user replica sets update when a node is deregistered or down
+ */
+const userReplicaSetNodes = async ({
+  numUsers = 10,
+  numCreatorNodes = 10,
+  iterations,
+  executeAll,
+  executeOne
+}) => {
+
+  // Make sure there are 4 creator nodes
```

Comment:
should clean these comments to not specify number 4 here and number 10 on line 36